infra/dockerfile.commit_retrieval

```diff
@@ -6,7 +6,7 @@ ENV PATH="${PATH}:/git-cinnabar"
 # libcurl4 is required by git-cinnabar.
 # patch is required by the commit classifier script.
```

Comment:
@PromiseFru, I just realized this comment is no longer applicable, could you submit a follow-up PR to remove it?

Reply:
Certainly! I'll get right on it.